ios-base/Extensions/FontExtension.swift

```diff
@@ -10,9 +10,12 @@ import UIKit
 import RSFontSizes
 
 extension UIFont {
-  var bold: UIFont { return withWeight(.bold) }
-  var semibold: UIFont { return withWeight(.semibold) }
-  var light: UIFont { return withWeight(.ultraLight) }
+  static let h1Regular: UIFont = .font(size: 32).withWeight(.regular)
+  static let h2Regular: UIFont = .font(size: 16).withWeight(.regular)
+  static let h3Regular: UIFont = .font(size: 15).withWeight(.regular)
+  static let h1Medium: UIFont = .font(size: 32).withWeight(.medium)
+  static let h2Medium: UIFont = .font(size: 16).withWeight(.regular)
+  static let h3Medium: UIFont = .font(size: 15).withWeight(.regular)
```

Comment:
Please move the font sizes to meaningful constants as well.

Reply:
I don't like to have another enum with more constants. We got almost all of them added in `Font.PointSize`. I'm wondering if wouldn't be better to modify the `RSFontSizes` library to use `proportional` as a function instead a case so we can use `Font.PointSize` constants (and the case `fixed` for the not added ones)  and ask then for the proportion as we seem to need in the line 36

Reply:
We shouldn't modify a dependency for something that is convenient to us. The purpose of proportional is to provide a flexible way of scaling the base point size passed by parameter. `Font.PointSize`  are simply to provide meaningful predefined sizes but doesn't mean that we can have our own PointSize system, either an enum, constants or any other.